package.json

```diff
@@ -7,12 +7,13 @@
     "build": "npm run build:cjs && npm run build:esm",
     "build:cjs": "lerna exec --parallel -- babel src -d dist --ignore $npm_package_config_babel_ignore --root-mode upward",
     "build:esm": "BABEL_TYPE=esm lerna exec --parallel -- babel src -d esm --configFile babel.config.js --ignore $npm_package_config_babel_ignore --root-mode upward",
+    "build:storybook": "build-storybook -o storybook_dist",
```

Comment:
noticed we are `build:storybook`-ing differently to the way we do it on [Simorgh](https://github.com/bbc/simorgh/blob/latest/package.json#L12)

we include a `-c` option which specifies configuration to be used. [Storybook options].(https://storybook.js.org/docs/configurations/cli-options/)

not too sure what this config is for and what impact it will have if we specify it 👀

Reply:
Good spot, I'b been inclined to not add this for now, as it wasn't present before. But one we should keep an eye on.